Makefile

```diff
@@ -28,10 +28,9 @@ endif
 ifeq ($(COVERAGE),1)
 	@go test -cover -coverprofile=$(GOPATH)/src/$@/c.out $@ -test.v
 	@go tool cover -html=$(GOPATH)/src/$@/c.out
-else
-	@echo "TESTING..."
-	@go test $@ -test.v
 endif
+	@echo "TESTING... $@"
+	@go test $@ -test.v
```

Comment:
hmm, well - it was already doing that, and then not running tests at all if we weren't tracking coverage!
But I did make it conditional: dd39623868f26e5d8e90a1faeb755894d6ebaf81